HashRegistrar.sol

```diff
@@ -47,34 +47,52 @@ contract Deed {
     address constant burn = 0xdead;
     uint public creationDate;
     address public owner;
-    
+    event OwnerChanged(address newOwner);
```

Comment:
I'm concerned these events would be difficult to listen for, since they're on each deed; can we sensibly restrict the events to the registrar contract?